http/client.go

```diff
@@ -146,3 +228,17 @@ func getQuery(req cmds.Request) (string, error) {
 
 	return query.Encode(), nil
 }
+
+func isConnRefused(err error) bool {
```

Comment:
👏

Reply:
I'd love to take the praise but I'm pretty sure I copied that from corehttp or so :(